packages/stockflux-core/src/services/ChildWindowLauncher.js

```diff
@@ -1,18 +1,44 @@
 import * as OpenfinApiHelpers from '../openfin-api-utils/openfinApiHelpers';
 
+const windowAlreadyExist = async (windowName, childWindows) => {
```

Comment:
Maybe this should be another openfinApiHelper? I think I might have seen this used elsewhere - most likely in useChildWindow. There's no need to update that one as it will be replaced eventually by `openfin-react-hooks` `useChildWindow`. Yet I think many apps might benefit from this functionality in the future. Feel free to push back if you don't agree.

Reply:
I think this will be a future PR if it's needed elsewhere. As there is an issue in the backlog to change `useChildWindow` to use the `openfin-react-hooks` version.